test/cmdlineTests/model_checker_solvers_smtlib2/err

```diff
@@ -1,7 +1,23 @@
 Warning: CHC: 1 verification condition(s) could not be proved. Enable the model checker option "show unproved" to see all of them. Consider choosing a specific contract to be verified in order to reduce the solving problems. Consider increasing the timeout per query.
+ --> model_checker_solvers_smtlib2/input.sol:2:1:
+  |
+2 | pragma solidity >=0.0;
+  | ^ (Relevant source part starts here and spans across multiple lines).
 
 Warning: CHC analysis was not possible. No Horn solver was available. None of the installed solvers was enabled.
+ --> model_checker_solvers_smtlib2/input.sol:2:1:
+  |
+2 | pragma solidity >=0.0;
+  | ^ (Relevant source part starts here and spans across multiple lines).
 
 Warning: BMC: 1 verification condition(s) could not be proved. Enable the model checker option "show unproved" to see all of them. Consider choosing a specific contract to be verified in order to reduce the solving problems. Consider increasing the timeout per query.
+ --> model_checker_solvers_smtlib2/input.sol:2:1:
+  |
+2 | pragma solidity >=0.0;
+  | ^ (Relevant source part starts here and spans across multiple lines).
 
 Warning: BMC analysis was not possible. No SMT solver (Z3 or CVC4) was available. None of the installed solvers was enabled.
+ --> model_checker_solvers_smtlib2/input.sol:2:1:
+  |
+2 | pragma solidity >=0.0;
+  | ^ (Relevant source part starts here and spans across multiple lines).
```

Comment:
I can see now why these had no location :) Printing this does seem a bit useless. Maybe we could deal with it in a more generic way - by not printing the source if it spans the whole contract and has multiple lines?

Alternatively - maybe the mechanism for cutting out repeated errors should just ignore errors without location?

Reply:
Yea I think I prefer the second idea. The first one might touch parts outside the smtchecker that I'm not aware of.